public/js/cohortStudent.js

```diff
@@ -2,50 +2,32 @@ const addStd = document.getElementById('addStd');
 const studentName = document.getElementById('studentName');
 const gitHubUserName = document.getElementById('gitHubUserName');
 const addStudentButton = document.getElementById('addStudentButton');
-const loading = document.getElementById('loading');
 const Url = window.location.href;
 const splitUrl = Url.split('/');
```

Comment:
You could also do:
```js
const [,,cohortId] = url.split('/');
```
But it's up to you, although I want you to rename your `Url` to `url`